### PR TITLE
828 record blood pressure in two values

### LINKED ISSUE #828: [ui][jsonschema] Record blood pressure in two values
Record systolic / diasystolic blood pressure values in the data. (If we still want a single text input we could parse the text into two values.)

Also See:

https://github.com/openmsupply/open-msupply/pull/771#discussion_r1018934949

### PR DESCRIPTION
**Pull request by roxy-dao**
Fixes #828

# 👩🏻‍💻 What does this PR do? 
- Record the systolic and diastolic values separately for blood pressure
Screen Shot 2023-03-03 at 11 45 25 AM
- Also moved BMI up, since it's calculated from height and weight.

# 🧪 How has/should this change been tested? 
- Go to an Care Encounter and see values... or change values

### CODE REVIEW

**Review by clemens-msupply — APPROVED** [review]
Great! thank you!

File: server/service/src/sync/program_schemas/hiv_care_encounter.json [hunk 1]
```diff
@@ -389,10 +385,18 @@
           "properties": {
             "bloodPressure": {
               "description": "75367002\tBlood pressure [mmHg]",
-              "examples": ["90/60", "120/80"],
-              "pattern": "^\\d{3}\\/\\d{2}$",
-              "title": "BP in mmHg",
-              "type": "string"
+              "properties": {
+                "diastolic": {
+                  "pattern": "^\\d{3}",
```
Comment (clemens-msupply) [inline server/service/src/sync/program_schemas/hiv_care_encounter.json:390]: Pattern doesn't apply for number... You could have maximum: 999 though

Comment (clemens-msupply) [reply server/service/src/sync/program_schemas/hiv_care_encounter.json:390]: + minimum

Comment (roxy-dao) [reply server/service/src/sync/program_schemas/hiv_care_encounter.json]: [157a795](https://github.com/openmsupply/open-msupply/pull/1267/commits/157a795849691aaa257927f70da39c85a1b5f58b) (:

**Pull request merged**